Sourcecode/private/mx/impl/NoteReader.cpp

```diff
@@ -43,6 +43,7 @@
 #include "mx/utility/StringToInt.h"
 
 #include <map>
+#include <mx/api/PitchData.h>
```

Comment:
```suggestion
#include "mx/api/PitchData.h"
```

CLion is not smart enough to use quotes?